package.json

```diff
@@ -10,7 +10,8 @@
     "test": "ng test",
     "test:ci": "ng test --watch=false --code-coverage",
     "lint": "ng lint",
-    "e2e": "ng e2e"
+    "e2e": "ng e2e",
+    "bundle-report": "ng build --stats-json && webpack-bundle-analyzer dist/ngx-starter/stats.json"
```

Comment:
For consistency, should this script be called "build:bundle-report" or something like that?

Reply:
I like it. I moved it to `build:bundle-report` and added a section into the README that describes each of the available NPM scripts so folks don't have to hunt through the `package.json` file.